projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -206,6 +213,13 @@ public static function load_etk_features_flags() {
 	 * Can be moved back to load_features() once the feature no longer exists in the ETK plugin.
 	 */
 	public static function load_etk_features() {
+		// Don't load on agency sites.
+		if ( is_fully_managed_agency_site() ) {
+			return;
+		}
+
+		// Don't load if the user is not a wpcom user on WP Admin.
+		// The features is still required on the frontend page regardless of the user.
```

Comment:
Could you give some example of such features?

Reply:
For example, we need to load the `view.js` for both `event-countdown` and `timeline` blocks. In addition, the `wpcom-global-styles` notices users to upgrade their site to apply global styles to visitors.